contracts/AbstractConference.sol

```diff
@@ -197,8 +265,8 @@ contract AbstractConference is Conference, GroupAdmin {
      * @param _maps The attendance status of participants represented by uint256 values.
      */
     function finalize(uint256[] calldata _maps) external onlyAdmin onlyActive {
-        uint256 totalBits = _maps.length * 256;
-        require(totalBits >= registered && totalBits - registered < 256, 'incorrect no. of bitmaps provided');
+        uint256 totalBits = _maps.length.mul(256);
+        require(totalBits >= registered && totalBits.sub(registered) < 256, 'incorrect no. of bitmaps provided');
```

Comment:
`totalBits >= registered` is unnecessary as `totalBits.sub(registered) < 256` does the same check.

Reply:
This is true